docs/src/app/(public)/(content)/react/utils/use-render/demos/render-callback/css-modules/index.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+import * as React from 'react';
+import { useRender } from '@base-ui-components/react/use-render';
+import styles from './index.module.css';
+
+type CounterState = {
+  odd: boolean;
+};
+
+type CounterProps = {
+  className?: string;
+  render?: useRender.RenderProp<CounterState>;
+};
+
+function Counter(props: CounterProps) {
+  const { render, className, ...otherProps } = props;
+  const [count, setCount] = React.useState(0);
+  const odd = count % 2 === 1;
+  const state = React.useMemo(() => ({ odd }), [odd]);
+
+  const { renderElement } = useRender({
+    render: render ?? <button type="button" />,
+    state,
+    props: {
+      ...otherProps,
+      className: `${className} ${styles.Button}`,
```

Comment:
```suggestion
      className: `${styles.Button} ${className}`,
```

PS: some people will realize about `undefined` ending up in the DOM if we didn't pass one `className` in the demo 😅

Reply:
Fixed.